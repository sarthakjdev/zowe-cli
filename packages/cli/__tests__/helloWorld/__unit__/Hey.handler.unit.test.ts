import { UNIT_TEST_ZOSMF_PROF_OPTS } from "../../../../../__tests__/__src__/mocks/ZosmfProfileMock";

describe("hello world handler", () => {
    it("should say hello", () => {
        const handlerReq = require("../../../src/helloWorld/Hey.handler");

        expect(handlerReq).toBeDefined();
        expect(handlerReq).toMatchSnapshot();
    });
});
