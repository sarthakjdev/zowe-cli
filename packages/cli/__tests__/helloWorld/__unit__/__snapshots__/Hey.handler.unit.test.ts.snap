// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello world handler should say hello 1`] = `
Object {
  "default": [Function],
}
`;
